src/modules/data/sagas.js

```diff
@@ -6,8 +6,10 @@ import { sagas as RSVPSagas } from '@moderntribe/tickets/data/blocks/rsvp';
 import { sagas as TicketSagas } from '@moderntribe/tickets/data/blocks/ticket';
 import { sagas as AttendeesSagas } from '@moderntribe/tickets/data/blocks/attendees';
 
-[
-	RSVPSagas,
-	TicketSagas,
-	AttendeesSagas
-].forEach( sagas => store.run( sagas ) );
+export default () => {
```

Comment:
we should probably do the same pattern in our other plugins

Reply:
Yeah In this way we execute sagas only once, as right now `data` is included independently and then included in the blocks that triggers the `sagas` watchers to be executed twice.